### ISSUE TITLE
PARAMETER UPDATE: 2025

### ISSUE THREAD

**Issue by MImmesberger** [iza-institute-of-labor-economics/gettsim#806]
The following list will be updated in the future.

✅ : Already on main
❌ : Not relevant this year
Check box: To implement / check

#### Typical parameters to be updated

- ✅  Mindestlohn (already on main; Minijob increases to 556€)
- [ ] Kinderfreibetrag (increases to 9,600€)
- [ ] Kinderzuschlag
- [ ] Mindestunterhalt (increases slightly in 2025)
- ❌  Bürgergeld Regelsatz (does not change in 2025)
- [ ] Beitragsbemessungsgrenze Krankenversicherung (66,150€)
- [ ] Beitragsbemessungsgrenze Rentenversicherung (8,050€ both in East and West)
- [ ] Mean Zusatzbeitrag KV (2.5% in 2025)
- [ ] Vorläufiges Durchschnittsentgelt Rentenversicherung (50,493€ in 2025)
- [ ] Aktueller Rentenwert (39,32€ in 2025)
- [ ] Bezugsgröße Rentenversicherung (3,745€)

#### Additional parameters for this year

- [ ] Wohngeld increases by roughly 15%
- [ ] Steuerfreibetrag (increases to 12,084€)
- [ ] Kindergeld increases to 255€
- [ ] Kindersofortzuschlag increases to 25€
- ✅  Elterngeld Einkommensgrenze Vorjahr increases (already on main)
- [ ] Beitragssatz Pflegeversicherung increases by 0.2 pp

**Comment by hmgaudecker** [comment]
Still missing those two items that are not ticked off?

**Comment by MImmesberger** [comment]
Yes, still missing. Can do that soon. [Parameters Wohngeld](https://www.recht.bund.de/bgbl/1/2024/314/VO.html); [Parameters Mindestunterhalt](https://www.recht.bund.de/bgbl/1/2024/359/VO.html).
